tests/admin/test_webauthn.py

```diff
@@ -0,0 +1,92 @@
+from .. import util
+import duo_client.admin
+from .base import TestAdmin
+
+
+class TestWebauthn(TestAdmin):
+    def test_get_webauthncredentials_with_params(self):
+        """ Test to get webauthn credentials with params.
+        """
+        response = list(self.client_list.get_webauthncredentials(limit=8))[0]
+        uri, args = response['uri'].split('?')
+
+        self.assertEqual(response['method'], 'GET')
+        self.assertEqual(uri, '/admin/v1/webauthn-credentials')
```

Comment:
This will also need to be updated to `/admin/v1/webauthncredentials` based on the above comment.